packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -844,7 +844,7 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
 
   // Waits one frame and cause an empty scroll event (zero delta pixels).
   //
-  // This allows the thumb to show immediately when isAlwaysShown is true.
+  // This allows the thumb to show or hide immediately when isAlwaysShown is true.
```

Comment:
```suggestion
  // This allows the thumb to show or hide immediately when isAlwaysShown is true.
```
Why would we want the scrollbar to hide if `isAlwaysShown` is true?

Reply:
the scrollable widget content change smaller and no longer scrollable

Reply:
the unit test show this situation I think.

Reply:
I was wondering if that was it. :) The comment should probably mention that.

Reply:
That is true! I will do that.